py/desimeter/posparams/plotter.py

```diff
@@ -0,0 +1,191 @@
+# -*- coding: utf-8 -*-
+"""
+Plot time series of positioner parameters.
+"""
```

Comment:
For the record: it is good that you separated the plotting functions from the algorithmic functions so that the algorithmic functions can be run on a batch node if needed without a X11/GUI connection.

Reply:
Good to know that that is a use case